src/index.html

```diff
@@ -1,6 +1,19 @@
+<!DOCTYPE html>
 <link rel="stylesheet" href="./styles/main.scss" />
 <script src="https://unpkg.com/formiojs@latest/dist/formio.full.min.js"></script>
 
+<!-- Horrible code only impacting old browsers -->
```

Comment:
Can you update the comment for what browsers this does fix

Reply:
This feature detects and patches browsers that don't have fetch and the second one is those that don't have htmlTemplateElement. This includes but is not limited to Internet Explorer.